server/lib/notifications.js

```diff
@@ -215,9 +215,62 @@ const notifyUpdateSubscribers = async activity => {
   const emailOpts = { from: activity.data.fromEmail };
   const update = await models.Update.findByPk(activity.data.update.id);
   const allUsers = await update.getUsersToNotify();
-  return notifySubscribers(allUsers, activity, emailOpts);
+  const modifiedActivity = replaceVideosByImagePreviews(activity);
+  return notifySubscribers(allUsers, modifiedActivity, emailOpts);
 };
 
+function replaceVideosByImagePreviews(activity) {
+  const iframePreviewRegex = /<iframe\s[^>]*src="([^"]+)"[^>]*><\/iframe>/gi;
+  activity.data.update.html = activity.data.update.html
+    .replace(iframePreviewRegex, (_, href) => {
+      const { service, id } = parseServiceLink(href);
+      const imgSrc = constructPreviewImageURL(service, id);
+      if (imgSrc) {
+        return `<img src="${imgSrc}" alt="${service} content">`;
+      } else {
+        return '';
+      }
+    })
+    .replace(new RegExp('</iframe><figcaption></figcaption></figure>', 'g'), '')
+    .replace(new RegExp('width="100%" height="394"', 'g'), '');
```

Comment:
I don't think we still need these two?

Reply:
Nope, you are right. That's my mistake. 😞